integration/entrypoint.sh

```diff
@@ -239,7 +239,7 @@ function start_multiple_containers_shared_daemon {
 function start_single_container_on_stargz {
     echo "testing $FUNCNAME"
     nerdctl_prune_images
-    reboot_containerd shared
+    reboot_containerd multiple
```

Comment:
Why does the recovered snapshotter start in multiple mode?

Reply:
See L 247. The recovered nydus-snapshotter is started in multiple mode

Reply:
Ok